src/test/java/ru/training/at/hw3/page/DifferentElementsPage.java

```diff
@@ -0,0 +1,69 @@
+package ru.training.at.hw3.page;
+
+import ru.training.at.hw3.utils.WaitActions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import javax.inject.Inject;
+import java.util.List;
+
+public class DifferentElementsPage {
+
+    private WaitActions waitActions;
+    private WebDriver webDriver;
+    private WebElement webElement;
+
+
+    @FindBy(css = ".label-checkbox:first-child > input")
+    private WebElement waterCheckbox;
+
+    @FindBy(css = ".label-checkbox:nth-last-child(2) > input")
+    private WebElement windCheckbox;
```

Comment:
Checkboxes could be represented as `List<WebElement>`

Reply:
I just don't need them all, so I chose to implement them individually according to KISS principle

Reply:
I am not sure about KISS principal here. Mostly it looks like as YAGNI :)